zipkin-example/src/main/java/com/example/helloworld/HelloWorldApplication.java

```diff
@@ -43,6 +45,9 @@ public void run(HelloWorldConfiguration configuration,
         final Client client = new ZipkinClientBuilder(environment, brave)
                 .build(configuration.getZipkinClient());
 
```

Comment:
Hmm, do you think we should be coupling Rx Java to this? Perhaps people won't want to use RxJava. Just a thought. I'll add a test case when I get a spare moment.